### PR TITLE
Load contracts from default directories

### PR DESCRIPTION
**Pull request by mitchelli**
Fixes https://github.com/aeternity/aeternity/issues/4320

This PR is supported by Æternity Foundation

### CODE REVIEW

File: apps/aecore/src/aec_fork_block_settings.erl [hunk 1]
```diff
@@ -344,7 +348,15 @@ extra_accounts_json_file() ->
     "extra_accounts_test.json".
 
 contracts_json_file() ->
```
Comment (hanssv) [inline apps/aecore/src/aec_fork_block_settings.erl:350]: Isn't this just a subset of the function below? I can't remember why we wan't the TEST one?

**Review by hanssv — APPROVED** [review]

File: apps/aecore/src/aec_fork_block_settings.erl (hunk 1 above)
Comment (dincho) [reply apps/aecore/src/aec_fork_block_settings.erl:350]: This filename is a legacy one ... back then one could only set accounts (and later contracts) via this file. The suffix "_test" I believe comes from the original usage for tests only, but at some point all "custom" network ID's has been using this.

Comment (dincho) [reply apps/aecore/src/aec_fork_block_settings.erl:350]: Ah, I see what your comment is about now .. nvm

Comment (mitchelli) [reply apps/aecore/src/aec_fork_block_settings.erl]: I've moved the functions out of test and made a generic function for creating the file name.

**Review by hanssv — APPROVED** [review]

**Review by ThomasArts — APPROVED** [review]

**Pull request merged**